package.json

```diff
@@ -15,7 +15,7 @@
     "maddox": "./bin/maddox.js"
   },
   "engines": {
-    "node": ">=5.0.0"
+    "node": ">=6.0.0"
```

Comment:
We wouldn't want to change this field as nobody using node 5 would be able to use this module anymore.  This field just indicates that it works in node 5 and above.